README.md

```diff
@@ -241,7 +241,7 @@ app.model({
   subscriptions: {
     emitWoofs: (send) => {
       // emit a woof every second
-      setInterval(() =>  send('print', { woof: 'meow?' }), 1000)
+      setInterval(() =>  send('print', { printWoofs: 'meow?' }), 1000)
```

Comment:
Sorry, changed the wrong thing! 😓 It should be `setInterval(() =>  send('printWoofs', { woof: 'meow?' }), 1000)`

The argument in send should match the effect name, right?